conf/sidecar.yaml

```diff
@@ -158,6 +158,11 @@ sidecar:
       # The minimum value allowed for the cluster lease claim process task implementation is 30 seconds
       execute_interval: 100s
 
+    # DnsResolver to use (org.apache.cassandra.sidecar.common.server.dns.DnsResolver). Currently supported
+    # values are 'default' and 'resolveToIp'. Default will resolve hostnames to addresses and addresses to
+    # hostnames whereas resolveToIp will only resolve hostnames to addresses.
+    dnsResolver: default
```

Comment:
can we use snake case here to be consistent with the rest of the yaml?

Reply:
Updated